extensions/email/src/main/java/org/mule/extension/email/api/sender/SenderOperations.java

```diff
@@ -60,7 +60,7 @@
      */
     public void send(@Connection SenderConnection connection,
                      @UseConfig SMTPConfiguration configuration,
-                     @Optional(defaultValue = PAYLOAD) EmailContent content,
```

Comment:
I'll add the default again, but my concern is about transforming whats incoming in the payload to an "EmailContent instance", this is not going to work now.